src/intl/en/page-ethereum-history-founder-and-ownership.json

```diff
@@ -0,0 +1,65 @@
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-12": "See the complete list of Ethereum upgrades",
+"page-ethereum-history-founder-and-ownership-when-did-ethereum-launch-description-13": "Check out the future Ethereum roadmap",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-1": "One of the most unique aspects of Ethereum is its ownership structure, or more accurately, its lack of traditional ownership. Unlike a regular company, Ethereum:",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-2": "<strong>has no CEO or central authority</strong>",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-3": "is <strong>not controlled by any single entity</strong> or organization",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-4": "<strong>does not have shareholders</strong> in the traditional sense",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-5": "Instead, Ethereum works as a decentralized network. It's <strong>maintained by thousands of independent computers</strong> (nodes) around the world. This decentralized model is at the core of Ethereum's design and value.",
+"page-ethereum-history-founder-and-ownership-who-owns-and-runs-ethereum-now-description-6": "Several key groups help with Ethereum's ongoing development and governance:",
+"page-ethereum-history-founder-and-ownership-ethereum-foundation": "1. The Ethereum Foundation",
+"page-ethereum-history-founder-and-ownership-ethereum-foundation-description-1": "The <strong>Ethereum Foundation is a non-profit organization</strong> that supports Ethereum and related technologies. While important, it does not control the network. The Foundation:",
+"page-ethereum-history-founder-and-ownership-ethereum-foundation-description-2": "manages funds to support Ethereum development",
+"page-ethereum-history-founder-and-ownership-ethereum-foundation-description-3": "provides grants to projects building on Ethereum",
+"page-ethereum-history-founder-and-ownership-ethereum-foundation-description-4": "organizes community events and educational initiatives",
+"page-ethereum-history-founder-and-ownership-ethereum-foundation-description-5": "coordinates research efforts",
+"page-ethereum-history-founder-and-ownership-core-developers": "2. Core Developers and Researchers",
+"page-ethereum-history-founder-and-ownership-core-developers-description-1": "A global community of developers and researchers contribute to Ethereum's code and design. These contributors propose, discuss, and implement improvements through an open, public process. Vitalik Buterin remains respected in the community. However, decisions are made through group agreement rather than by any single person.",
```

Comment:
```suggestion
"page-ethereum-history-founder-and-ownership-core-developers-description-1": "A global community of developers and researchers contribute to Ethereum's code and design. These contributors propose, discuss, and implement improvements through an open, public process. Vitalik Buterin remains respected in the community, however, decisions are made through group agreement rather than by any single person.",
```